torch/autograd/profiler_util.py

```diff
@@ -127,11 +127,18 @@ def _populate_cpu_children(self):
                         # this can't be a parent
                         current_events.pop()
                     else:
-                        parent.append_cpu_child(event)
-                        assert (
-                            event.cpu_parent is None
-                        ), f"There is already a CPU parent event for {event.key}"
-                        event.set_cpu_parent(parent)
+                        not_overlaped = True
```

Comment:
nit: should be not_overlapped

Reply:
Thanks for your heads up, it's been changed